internal/xds/bootstrap/bootstrap.go

```diff
@@ -83,6 +84,40 @@ func (cc ChannelCreds) String() string {
 	return cc.Type + "-" + string(b)
 }
 
+// CallCredsConfig contains the call credentials configuration to be used on
+// RPCs to the management server.
+type CallCredsConfig struct {
+	// Type contains a name identifying the call credentials type.
+	Type string `json:"type,omitempty"`
+	// Config contains the JSON configuration for this call credentials.
```

Comment:
I think it's worth mentioning in the godoc that this field is optional. From [A97](https://github.com/grpc/proposal/blob/master/A97-xds-jwt-call-creds.md#configuring-call-credentials-in-xds-bootstrap-file):
> // The "config" field is optional; it may be missing if the
// credential type does not require config parameters.